src/azure-cli/azure/cli/command_modules/vm/_vm_utils.py

```diff
@@ -602,6 +602,23 @@ def trusted_launch_warning_log(namespace, generation_version, features):
             logger.warning(UPGRADE_SECURITY_HINT)
 
 
+def validate_vm_disk_trusted_launch(namespace, disk_security_profile):
+    if disk_security_profile is None:
+        return
+
+    from ._constants import UPGRADE_SECURITY_HINT
```

Comment:
The warning message mentioned in the feature request is:
`Consider upgrading security for your workloads using Azure Trusted Launch VMs. To know more about Trusted Launch, please visit https://aka.ms/TrustedLaunch`

Hence the link in the UPGRADE_SECURITY_HINT field should be changed from "https://learn.microsoft.com/en-us/azure/virtual-machines/trusted-launch" to "https://aka.ms/TrustedLaunch".